dependencies.js

```diff
@@ -1,3 +1,7 @@
-export * as http from 'https://deno.land/x/http@61a3911be7e01273e13bf35a3a16285f413f0b70/mod.ts';
-export { Status as status } from 'https://deno.land/x/http@61a3911be7e01273e13bf35a3a16285f413f0b70/http_status.ts';
-export { STATUS_TEXT as statusText } from 'https://deno.land/x/http@61a3911be7e01273e13bf35a3a16285f413f0b70/http_status.ts';
+export * as http from "https://deno.land/x/http@61af419bbc5717c2e2552050aacb20ef1b17480b/server.ts";
```

Comment:
I guess this is the line that fixes the README example, right? Any breaking changes I should be aware of?

Reply:
`deno_std` doesn't have a changelog. But [deno's changelog](https://github.com/denoland/deno/releases) for 0.3.4 mentions that the deno module is gone and replaced by a Deno global var. Other than that it's hard to say.